docs/automated-process.md

```diff
@@ -71,13 +71,18 @@ Note that most of the following items cannot be done by yourself, but need someo
     The notification includes a link to the page where the upload step can be
     approved.
 
-2. Check out the PR in your `ansible-build-data` clone and copy the updated
-   porting guide from its `${MAJOR_VERSION}` directory into the
-   [`docs/docsite/rst/porting_guides/`](https://github.com/ansible/ansible-documentation/tree/devel/docs/docsite/rst/porting_guides/)
-   directory of the [`ansible-documentation`
-   repository](https://github.com/ansible/ansible-documentation/). Create a
-   PR for updating the porting guide for the `devel` branch of
-   `ansible-documentation`.
+2. If a porting guide exists in the `ansible-build-data` PR, trigger [the automated
+   workflow](https://github.com/ansible/ansible-documentation/actions/workflows/release-porting-guide.yml)
+   on the **Actions** tab of the `ansible-documentation` repository to create the porting guide PR. This workflow has the following inputs:
+
+    * **Release Branch name**. Specify the name of the branch from the newly created [ansible-build-data](https://github.com/ansible-community/ansible-build-data/) PR for the release, for example: `refs/pull/576/merge`. This branch refers to the current state of Pull Request #576 in the ansible-build-data repository. You need to adjust the number to the PR's number.
```

Comment:
I feel like I'm fundamentally missing something here. I think I already commented that all the most recent branch names are something like `publish-12.0.0b1` . So why is the branch name here `refs/pull/<PR>/merge`?  Also looking at recent porting guide creation runs, they end in `head` instead of `merge`.

Not saying these instructions are incorrect. I'm just trying to understand why the branch name isn't ... the branch name showed in the GitHub UI, and why it says merge here and head in the recent runs of the workflow.